### ISSUE TITLE
Can't load subscriptions

### ISSUE THREAD

**Issue by mbreese** [MicrosoftDocs/vsonline#526]
I'd like to try VSO, but when I try to create a new Codespace Plan, the online tool fails to find my Azure subscription. I'm not quite sure what is going on...

Note: I just created my Azure account, but I've been able to create new VMs, etc through the azure portal. I don't know why the subscription isn't appearing for VSO.

I'm using Chrome on Mac Catalina, but I also tried using MS Edge in a Windows VM (from Azure!)

**Comment by jkeech** [comment]
@mbreese, I think you are running into #39. We are working on supporting more tenants than the default tenant, but there's currently not a good workaround until that is in place ☹️.

**Comment by mbreese** [comment]
Wow was that a chase...

I'm still pretty new to Azure, so not quite tied into the full Azure Active Directories, etc... (I'm fairly familiar with LDAP though).

I could confirm that VSO was getting a list of multiple tenants by looking at the requests made in the Chrome network inspector. Calls to https://management.azure.com/tenants?api-version=2019-05-10 showed multiple tenants.

Turned out that was exactly the issue, but not for an obvious reason. My email account was added to a mailing list managed by my kid's school. This was the same email account as my Microsoft account. This caused my Microsoft account to be associated with the school's AD, without my knowledge of this ever happening. Because I added my Azure account after this, my primary tenant was the school's directory, not my own AAD (with the associated Azure subscription).

Following through the directions here: https://docs.microsoft.com/en-us/azure/active-directory/b2b/leave-the-organization I was able to remove myself from this email list/group. Once I did that, VSO found my subscription easily.

I leave the extra comments to maybe help the next person.
